python_modules/dagster/dagster/core/selector/subset_selector.py

```diff
@@ -29,13 +33,20 @@ class OpSelectionData(
     Attributes:
         op_selection (List[str]): The queries of op selection.
         resolved_op_selection (AbstractSet[str]): The names of selected ops.
-        ignored_solids (List[Node]): The solids in the original full graph but outside the current
-            selection. This is used in run config resolution to handle unsatisfied inputs correctly.
+        ignored_solids_dict (Dict[Union[Node, str], Dict]): The nodes in the original full graph but
+            outside the current selection. Nested selection supported.
+            This is used in run config resolution to handle unsatisfied inputs and config mapping correctly.
```

Comment:
what exactly is the keying scheme / structure of this? Not obvious to me from just the type sig

Reply:
updated